lib/hammer_cli_csv/sync_plans.rb

```diff
@@ -0,0 +1,122 @@
+# Copyright 2013-2014 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+
+module HammerCLICsv
+  class CsvCommand
+    class SyncPlansCommand < BaseCommand
+      command_name 'sync-plans'
+      desc         'import or export repository sync plans'
+
+      option %w(--organization), 'ORGANIZATION', 'Only process organization matching this name'
```

Comment:
i'm seeing this all over the place. would you be able to create a base csv command that contains all the similar code?